src/Tools/GetDocumentInsider/src/Commands/GetDocumentCommandWorker.cs

```diff
@@ -206,18 +208,22 @@ private bool GetDocuments(IServiceProvider services)
         var getDocumentsMethod = GetMethod(
             GetDocumentsMethodName,
             serviceType,
-            GetDocumentsParameterTypes,
-            GetDocumentsReturnType);
+            _getDocumentsParameterTypes,
+            _getDocumentsReturnType);
         if (getDocumentsMethod == null)
         {
             return false;
         }
 
+        var generateWithVersionMethod = serviceType.GetMethod(
```

Comment:
We're not using the `GetMethod` helper because we don't want to report errors?  For back compat, I guess we're going to ignore it if it's present but not in the shape we want?

Reply:
Correct.